docs/source/conf.py

```diff
@@ -49,6 +49,7 @@
     "sphinx.ext.doctest",
     "sphinx.ext.ifconfig",
     "sphinx.ext.viewcode",
+    "sphinx_search.extension",
```

Comment:
"Search as you type" functionality on Read the Docs is now provided through their new JavaScript add-ons https://github.com/readthedocs/addons so we shouldn't need to change the Sphinx extensions.

These are the steps: https://github.com/kedro-org/kedro/issues/2798#issuecomment-1781302147

so in principle this falls completely on the front-end side

Reply:
Also notice that add-ons need to be enabled, so to test this I recommend setting up a separate Read the Docs project and then go to

https://beta.readthedocs.org/dashboard/{project-name}/addons/edit/

![image](https://github.com/kedro-org/kedro/assets/316517/df22c098-2dc3-4432-ae79-e768f8e88373)